pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -52,30 +57,34 @@ edk2.mkDerivation projectDscPath (finalAttrs: {
     cp ${seabios}/Csm16.bin OvmfPkg/Csm/Csm16/Csm16.bin
   '';
 
-  postFixup = if stdenv.hostPlatform.isAarch then ''
+  # FIXME: remove me when https://github.com/tianocore/edk2/pull/4275 lands in a stable release.
+  patches = [
+    ./0001-RiscVVirt-support-split-code-and-vars-binaries.patch
+  ];
+
+  postFixup =
+  let
+    shortName = builtins.elemAt (lib.splitString "/" suffix) 1;
+  in ''
     mkdir -vp $fd/FV
-    mkdir -vp $fd/AAVMF
+    mkdir -vp $fd/${suffix}
```

Comment:
I'm not really satisfied about this mess, I will probably suggest a cleanup to upstream.